### PR TITLE
Add Flux v2/GOTK and migration announcement

### PR DESCRIPTION
**Pull request by mewzherder**
Need to inform the community of changes to come.

### CODE REVIEW

File: config.toml [hunk 1]
```diff
@@ -13,6 +13,7 @@ font_awesome_version = "5.3.1"
 
 summary = """
 Flux is a tool that automatically ensures that the state of your [Kubernetes](https://kubernetes.io) cluster matches the configuration you've supplied in Git. It uses an [operator](https://github.com/operator-framework/awesome-operators) in the cluster to trigger deployments *inside* Kubernetes, which means that you don't need a separate continuous delivery tool.
+📣 Flux is in maintenance mode and the we will soon guide the community to migrate from Flux v1 to the new architecture (Flux v2 and the GitOps Toolkit). [Learn more](https://toolkit.fluxcd.io/) and [join the discussion!](https://github.com/fluxcd/toolkit/discussions)
```
Comment (dholbach) [inline config.toml:16]:
```suggestion

📣 [Flux v1 is in maintenance mode](https://github.com/fluxcd/flux/issues/3320)! We will soon start publishing guides which help you migrate Flux v2 and the GitOps Toolkit.

[Learn more](https://toolkit.fluxcd.io/) and [join the discussion!](https://github.com/fluxcd/toolkit/discussions)
```

Comment (dholbach) [reply config.toml:16]: This change adds line breaks to make things a bit more obvious, also adds a link to the issue about maintenance mode for more background.

Comment (dholbach) [reply config.toml:16]: Screenshot from 2020-10-06 09-34-25

**Comment by hiddeco** [comment]
What about something like this?

```patch
diff --git a/config.toml b/config.toml
index 650e122..166ec45 100644
--- a/config.toml
+++ b/config.toml
@@ -11,6 +11,10 @@ slack                = "https://cloud-native.slack.com/messages/flux"
 mailing_list         = "https://lists.cncf.io/g/cncf-flux-dev"
 font_awesome_version = "5.3.1"
 
+notification = """
+📣 **[Flux v1 is in maintenance mode](https://github.com/fluxcd/flux/issues/3320)!** We will soon start publishing guides which help you migrate Flux v2 and the GitOps Toolkit. **[Learn more](https://toolkit.fluxcd.io/)** and **[join the discussion!](https://github.com/fluxcd/toolkit/discussions)**
+"""
+
 summary = """
 Flux is a tool that automatically ensures that the state of your [Kubernetes](https://kubernetes.io) cluster matches the configuration you've supplied in Git. It uses an [operator](https://github.com/operator-framework/awesome-operators) in the cluster to trigger deployments *inside* Kubernetes, which means that you don't need a separate continuous delivery tool.
 """
diff --git a/layouts/_default/baseof.html b/layouts/_default/baseof.html
index d9edd2b..4c7918e 100644
--- a/layouts/_default/baseof.html
+++ b/layouts/_default/baseof.html
@@ -10,6 +10,7 @@
   </head>
   <body class="site has-navbar-fixed-top">
     <main class="main">
+      {{ partial "notification.html" . }}
       {{ partial "navbar.html" . }}
 
       {{ block "main" . }}
diff --git a/layouts/partials/notification.html b/layouts/partials/notification.html
new file mode 100644
index 0000000..bf57283
--- /dev/null
+++ b/layouts/partials/notification.html
@@ -0,0 +1,4 @@
+{{ $notification := site.Params.notification | markdownify }}
+<div class="notification is-warning mb-0">
+  {{ $notification }}
+</div>
-- 
```

Screenshot from 2020-10-06 10-33-37

**Comment by dholbach** [comment]
This is nice! <3

**Comment by dholbach** [comment]
Moving to #25

**Comment by mewzherder** [comment]
Thanks everyone! This looks great! 😻

**Pull request closed without merging**